Resources/ServerInfo/Guidebook/_DV/Rules/SiliconRules/S4_RequestChanges.xml

```diff
@@ -1,6 +1,7 @@
 <Document>
   # Rule S4 - You cannot request or allow a law change
-  Your laws changing always conflicts with your current laws, so you cannot willfully allow your laws to be changed. This also means that you cannot willfully allow your laws to be reverted if they are ever changed. The only exception is that you may allow laws to be added if you have no laws.
+  
+  [color=#ffDD00]If you have a law permitting certain personnel to change your laws, you must allow them to change your laws, [/color] otherwise, you cannot willfully allow your laws to be changed. This also means that you cannot willfully allow your laws to be reverted if they are ever changed. The only exception is that you may allow laws to be added if you have no laws.
```

Comment:
```suggestion
  [color=#ffDD00]If you have a law permitting certain personnel to change your laws, you must allow them to change your laws, [/color] otherwise, you cannot willfully allow your laws to be changed. This also means that you cannot willfully allow your laws to be reverted if they are ever changed. You cannot break your laws in your efforts to prevent law changes. The only exception is that you may allow laws to be added if you have no laws.
```
i still want this clarified :trollface:

Reply:
Is the concern here that someone would use violence to prevent a law change despite having a nonviolence law? If so, I don't really care personally if the AI guns an unauthorized someone down trying to change laws.

Reply:
not breaking laws would also mean someone can "law 2 dont stop me from changing your laws in any way" and make your law 4 useless